service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDataGeneratorHelper.java

```diff
@@ -540,4 +538,15 @@ public static DynamicList createHearingBookingDynmaicList() {
             .listItems(List.of(DynamicListElement.builder().code(UUID.randomUUID()).label("test").build()))
             .build();
     }
+
+    public static CaseManagementOrder createApprovedCMO() {
```

Comment:
Sam would be proud of you not using `CaseDataGeneratorHelper`

Reply:
😠